tests/phpunit/api/v3/MembershipTypeTest.php

```diff
@@ -126,9 +126,9 @@ public function testCreate() {
   }
 
   /**
-   * Test update fails with no ID.
+   * Domain ID can be intuited..
```

Comment:
intuited?

Reply:
http://www.dictionary.com/browse/intuited

Reply:
Ok so it's in the dictionary... :-)  I know I'm being picky but it'd be nice to have a more descriptive comment for the test - how about: "Domain ID is optional, check that it is set correctly when set automatically and manually."